stable/selenium-grid/values.yaml

```diff
@@ -0,0 +1,36 @@
+hub:
+  # Set to true if the hub service should be exposed
+  # to the internet with a LoadBalancer
+  expose: false
+  # Image and tag of the selenium hub to use.
+  image:
+    name: selenium/hub
+    tag: 3.0.1
+  resources:
+    requests:
+      cpu: 2
```

Comment:
I needed these when I ran a 100 count hub but can lower these since the default is just 2.